aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -337,13 +337,18 @@ Tensor view_batching_rule(const Tensor& self, IntArrayRef size) {
 }
 
 template <typename F, F Func, typename... ExtraArgs>
-Tensor unary_pointwise_batching_rule(const Tensor& input, ExtraArgs... args) {
+Tensor unwrap_and_call(const Tensor& input, ExtraArgs... args) {
   auto* input_batched = unsafeGetBatchedImpl(input);
   auto output_physical = Func(input_batched->value(), args...);
   auto old_bdims = input_batched->bdims();
   return makeBatched(output_physical, BatchDims(old_bdims.begin(), old_bdims.end()));
 }
 
+template <typename F, F Func, typename... ExtraArgs>
```

Comment:
I am not sure to understand why you need this extra function? How is it different from the one above?

Reply:
The only thing that is different is th ename. I thought it would be strange if we used "unary_pointwise_batching_rule" to implement batching rules for operations that aren't unary pointwise (e.g., the complex view functions aren't unary pointwise).

Reply:
But I guess I could just name "unary_pointwise_batching_rule" to "unwrap_and_call"

Reply:
Ho ok. I was looking very hard for some subtle template differences.
I guess that using `unwrap_and_call` is better.